nrich-search/src/main/java/net/croz/nrich/search/util/PathResolvingUtil.java

```diff
@@ -18,6 +18,10 @@ public static String[] convertToPathList(String path) {
         return path.split(PATH_REGEX);
     }
 
+    public static String joinPath(String... pathList) {
```

Comment:
It seems we are missing a test for this method. In fact, we are missing tests for the entire util.

Reply:
I can add it, although not sure it is worth it, it is tested just not explicitly with a dedicated test.